core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -1762,6 +1762,8 @@ public SqlSyntax getSyntax() {
 
   public static final SqlFunction NULLIF = new SqlNullifFunction();
 
+  public static final SqlFunction REGEXP_SUBSTR = new SqlRegexpSubstrFunction();
+
```

Comment:
Does the `REGEXP_SUBSTR` belongs to standard SQL ? I remember the `PostgreSql` does.

Reply:
You can add it to `SqlLibraryOperators` under the `MySql` and `Oracle` sql dialects.